lib/index.js

```diff
@@ -2,10 +2,12 @@ var IdentityStore = require('./identity_store');
 var Hub = require('./hub');
 var StageBasedMessaging = require('./stage_based_messaging');
 var MessageSender = require('./message_sender');
+var utils = require('./utils.js');
 
 module.exports = {
     IdentityStore: IdentityStore,
     Hub: Hub,
     StageBasedMessaging: StageBasedMessaging,
-    MessageSender: MessageSender
+    MessageSender: MessageSender,
+    utils: utils
```

Comment:
i would leave it as is, camelcase used for 'classes' and lowercase for collection of functions in util.js